pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -2915,7 +2915,11 @@ The max allowed delay for delayed delivery (in milliseconds). If the broker rece
     private boolean exposeTopicLevelMetricsInPrometheus = true;
     @FieldContext(
             category = CATEGORY_METRICS,
-            doc = "If true, export buffered metrics"
+            doc = "Set to true to enable the broker to cache the metrics response, default is false. "
+                    + "For scraping metrics more than once per scrape period(defined by "
+                    + "`managedLedgerStatsPeriodSeconds`, please make sure `managedLedgerStatsPeriodSeconds`"
+                    + " same with your time series DB scrape interval.), the broker generates metrics at the first "
+                    + "scrape and returns the same response for the rest of the scrape period. "
```

Comment:
@dao-jun  I'd simplify the description as follows:
```suggestion
            doc = "Set to true to enable the broker to cache the metrics response; the default is false. "
                    + "The caching period is defined by `managedLedgerStatsPeriodSeconds`. "
                    + "The broker returns the same response for subsequent requests within the same period. "
                    + "Ensure that the scrape interval of your monitoring system matches the caching period."
```